src/database/migrations/20200925034242-assessments-setup.ts

```diff
@@ -0,0 +1,457 @@
+import { QueryInterface, DataTypes } from 'sequelize';
+
+export default {
+    up: async (queryInterface: QueryInterface): Promise<void> => {
+        // Transactions are automatically use because a namespace is injected into sequelize when fetching configurations
+        await queryInterface.sequelize.transaction(async () => {
+            /**
+             * course_question_assessment_info
+             */
+            await queryInterface.createTable('course_question_assessment_info', {
+                id: {
+                    field: 'course_question_assessment_info_id',
+                    type: DataTypes.INTEGER,
+                    autoIncrement: true,
+                    primaryKey: true,
+                },
+                courseWWTopicQuestionId: {
+                    field: 'course_topic_question_id',
+                    type: DataTypes.INTEGER,
+                    allowNull: false,
+                    // Defaults to cascade when generating the db, no action on migrations
+                    onUpdate: 'CASCADE',
+                    // Different from sequelize model, on models I do this with associations so I can use types
+                    references: {
+                        model: 'course_topic_question',
+                        key: 'course_topic_question_id',
+                    },
+                },
+                randomSeedSet: {
+                    field: 'course_question_assessment_info_random_seed_set',
+                    type: DataTypes.ARRAY(DataTypes.INTEGER),
+                    allowNull: false,
+                    defaultValue: []
+                },
+                additionalProblemPaths: {
+                    field: 'course_question_assessment_info_additional_problem_paths',
+                    type: DataTypes.ARRAY(DataTypes.TEXT),
+                    allowNull: false,
+                    defaultValue: []
+                },
+                active: {
+                    field: 'course_question_assessment_info_active',
+                    type: DataTypes.BOOLEAN,
+                    allowNull: false,
+                    defaultValue: true,
+                },
+                updatedAt: {
+                    field: 'updated_at',
+                    type: DataTypes.DATE,
+                    allowNull: false,
+                },
+                createdAt: {
+                    field: 'created_at',
+                    type: DataTypes.DATE,
+                    allowNull: false,
+                },
+            });
+
+            /**
+             * topic_assessment_info
+             */
+            await queryInterface.createTable('topic_assessment_info', {
```

Comment:
Is there a reason we dropped the `course_` prefix from this but not from the `course_question_assessment_info`?